src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import sentry_sdk
+
+from rest_framework.response import Response
+from rest_framework.exceptions import ParseError
+
+from sentry import features, eventstore
+from sentry.api.bases import OrganizationEventsEndpointBase, NoProjects
+from sentry.snuba import discover
+
+
+logger = logging.getLogger(__name__)
+
+
+def serialize_event(event, parent, is_root=False):
+    return {
+        "event_id": event["id"],
+        "span_id": event["trace.span"],
+        "transaction": event["transaction"],
+        "project_id": event["project_id"],
+        "parent_event_id": parent,
+        "is_root": is_root,
+    }
+
+
+class OrganizationEventsTraceLightEndpoint(OrganizationEventsEndpointBase):
+    def has_feature(self, organization, request):
+        return features.has(
+            "organizations:trace-view-quick", organization, actor=request.user
+        ) or features.has("organizations:trace-view-summary", organization, actor=request.user)
+
+    def get(self, request, organization, trace_id):
+        if not self.has_feature(organization, request):
+            return Response(status=404)
+
+        try:
+            params = self.get_snuba_params(request, organization)
+        except NoProjects:
+            return Response([])
+
+        with self.handle_query_errors():
+            result = discover.query(
+                selected_columns=[
+                    "id",
+                    "timestamp",
+                    "transaction",
+                    "project_id",
+                    "trace.span",
+                    "trace.parent_span",
+                    'to_other(trace.parent_span, "", 0, 1) AS root',
+                ],
+                # We want to guarantee at least getting the root, and hopefully events near it with timestamp
+                # id is just for consistent results
+                orderby=["-root", "-timestamp", "id"],
+                params=params,
+                query=f"event.type:transaction trace:{trace_id}",
+                limit=100,
+                referrer="api.trace-view.get_ids",
+            )
+            if len(result["data"]) == 0:
+                return Response(status=404)
+
+        event_id = request.GET.get("event_id")
+        if event_id is None:
+            raise ParseError("Only the light trace view is supported at this time")
+        is_root = lambda item: item["root"] == "1"
```

Comment:
Nit: Since it's being assigned a name, why not just use regular function syntax with `def is_root(...)`?